src/Microsoft.OData.Core/UriParser/Resolver/ODataUriResolver.cs

```diff
@@ -151,12 +169,22 @@ public virtual IEdmTerm ResolveTerm(IEdmModel model, string termName)
 
             IList<IEdmTerm> results = FindAcrossModels<IEdmTerm>(model, termName, /*caseInsensitive*/ true);
 
+            if (results == null)
+            {
+                return null;
+            }
+
+            if (results == null)
+            {
+                return null;
+            }

```

Comment:
what's the purpose to test Results twice?

Reply:
Merge error. 

---
In reply to: [370388471](https://github.com/OData/odata.net/pull/1602#discussion_r370388471) [](ancestors = 370388471)